docs/02-Install.md

```diff
@@ -167,29 +168,33 @@ More detailed specifications can be consulted in the
 - `/data/*` subdirectories were created by the imaging install script. If not,
     it may be due to `root:root` ownership of the `/data/` mount on your
     system. Ensure these subdirectories are created manually, particularly:
-    `/data/$PROJ/data/*`, `/data/$PROJ/bin/mri/` and `/data/incoming/`
+    `/data/$PROJECT/bin/mri/`, `/data/incoming/`, and those inside 
+    `/data/$PROJECT/data/`, namely `trashbin`, `tarchive`, `pic`, `logs`, 
+    `jiv`, `assembly`, and `batch_output`.
```

Comment:
@cmadjar 
changed the order here too as per your earlier comment on alphabetical order